doc/xml/user-guide.xml

```diff
@@ -416,13 +416,13 @@
 
                 <execute filter="n" output="y">
                     <exe-cmd>{[project-exe]} info</exe-cmd>
-                    <exe-highlight>(oldest|latest) backup label</exe-highlight>
+                    <exe-highlight>(full|incr|diff) backup</exe-highlight>
                 </execute>
             </execute-list>
 
-            <p>The oldest and newest backups are shown in the info output.  The oldest backup will <i>always</i> be a full backup (indicated by an <id>F</id> at the end of the label) but the newest backup can be full, differential (ends with <id>D</id>), or incremental (ends with <id>I</id>).</p>
+            <p>The backups are displayed oldest to newest.  The oldest backup will <i>always</i> be a full backup (indicated by an <id>F</id> at the end of the label) but the newest backup can be full, differential (ends with <id>D</id>), or incremental (ends with <id>I</id>).</p>
 
-            <p>More information about the <cmd>backup</cmd> command can be found in the <link section="/backup">Backup</link> section.</p>
+            <p>More information about the <cmd>backup</cmd> command can be found in the <link section="/backup">Backup</link> section. More information about the <cmd>info</cmd> command can be found in the <link section="/pitr">Point-in-Time Recovery</link> section.</p>
```

Comment:
I think it makes sense to break info off into a separate section and bring the fields details from PITR up into that section.

See: https://github.com/pgbackrest/pgbackrest/commit/65f229b44f48c31a187bfd17edd0d9fc7f3c65a7